src/bp/core/services/ghost/db-driver.ts

```diff
@@ -127,7 +129,10 @@ export default class DBStorageDriver implements StorageDriver {
     }
   }
 
-  async directoryListing(folder: string): Promise<string[]> {
+  async directoryListing(
+    folder: string,
+    options: { excludes?: string | string[] } = { excludes: [] }
```

Comment:
is there a reason why we allow excludes to be a string? forcing it to be an array would simplify the code but making it breaking ?

Reply:
I would also only allowed a list of string, but it was that way from the beginning , is part of the SDK and the disk driver is implemented the same way